pytest.ini

```diff
@@ -7,4 +7,5 @@ markers =
     notebook: testing notebooks can be slow. But needs to be performed to ensure that the documention is tested.
     no_cache_init: a test where the cache is not initialised. Must be run with --forked.
     no_eccodes: a test which should pass when ecCodes is not installed
+    with_proj: a test which should only pass if PROJ is installed and compatible with your system
```

Comment:
I wonder if we can detect it automatically as we do for other modules in `testsing.py, e.g. for pyfdb.

Reply:
I created the conda env on atos with the standard yaml above and got the error I reported in #177.

Reply:
The ci tests are not running the `with_proj` test automatically. Please modify the`unit-tests` in `Makefile` to make it work.

Reply:
I think the ci jobs are running the test, unless I am misunderstanding what you mean. For me they run fine locally with `make unit-tests` and the github actions see them and run them:

![image](https://github.com/ecmwf/earthkit-data/assets/53045993/803f686c-c4ea-4080-acfd-78ee1da1fd3f)

Reply:
I am sorry you are right.